rest/blip_sync.go

```diff
@@ -205,39 +231,44 @@ func (bh *blipHandler) handleSetCheckpoint(rq *blip.Message) error {
 // Received a "subChanges" subscription request
 func (bh *blipHandler) handleSubscribeToChanges(rq *blip.Message) error {
 
-	// Depeding on the db sequence type, use correct zero sequence for since value
-	since := bh.db.CreateZeroSinceValue()
+	subChanges := newSubChanges(rq, bh.blipSyncContext, bh.db.CreateZeroSinceValue(), bh.db.ParseSequenceID)
```

Comment:
Are the new helpers really just request parameter helpers?  If so, could we rename to something like subChangesParams?  I think it would make their usage more transparent here.

Reply:
Yeah, that's basically what they are.  I renamed them all accordingly.

Reply:
Another naming possibility would be:

- subChangesRequest
- revRequest 
- etc..